manifest.json

```diff
@@ -3,10 +3,11 @@
 	"author": "Ghostery",
 	"name": "__MSG_name__",
 	"short_name": "Ghostery",
-	"version": "8.5.3",
-	"version_name": "8.5.3",
+	"version": "8.5.4",
+	"version_name": "8.5.4",
 	"default_locale": "en",
 	"description": "__MSG_short_description__",
+	"debug": true,
```

Comment:
Should this be here?

Reply:
yea it's fine for `develop`. It gets removed by the builder.